phenol-annotations/src/main/java/org/monarchinitiative/phenol/annotations/formats/hpo/HpoDisease.java

```diff
@@ -96,6 +97,17 @@ default Stream<HpoDiseaseAnnotation> phenotypicAbnormalitiesStream() {
     return StreamSupport.stream(Spliterators.spliterator(phenotypicAbnormalities(), phenotypicAbnormalitiesCount(), Spliterator.DISTINCT & Spliterator.SIZED), false);
   }
 
+  /**
+   * The method is kept for backward compatibility. However, the method throws {@link PhenolRuntimeException} upon each invocation.
```

Comment:
There is not much point in keeping deprecated methods since it is only our group that is using phenol. I think it would be better to just modernize and cleanup and we will know how to adjust our application code!